requirements.txt

```diff
@@ -1,16 +1,16 @@
-awkward~=2.0.0
+awkward~=2.1.0
 boost-histogram~=1.3.0
 decaylanguage~=0.15.0
 hepstats~=0.6.0
 hepunits~=2.3.0
 hist~=2.6.0
 histoprint~=2.4.0
-iminuit~=2.18.0
-matplotlib>2.0.0
-mplhep~=0.3.0
+iminuit~=2.21.0
+matplotlib>3.4.0
+mplhep~=0.3.26
 numpy>=1.13.1
 particle~=0.21.0
 pylhe~=0.5.0
-resample~=1.5.0
+resample~=1.6.0
```

Comment:
Ah I see `resample` `v1.6.0` is Python 3.8+ only. Should we keep `resample` at `v1.5.x`, do this PR, make a new `scikit-hep` release, and then start working on a Python 3.8+ version of `scikit-hep`?

Reply:
No strong feeling. We can also move forward removing 3.7 for a new "major release", cf. the discussion you started about using CalVer instead of SemVer. I would hope that very few people are doing actualy physics analysis on 3.7 these days!

Reply:
Actually, let's be safe and still make a new (minor) release with Python 3.7 since uproot 5 only made it to the last scikit-hep release. Then we move to 3.8+ and most likely to CalVer versioning.

Merge this if happy too and we then move forward. Thanks a lot.

Reply:
OK, release 5.1.0 is the last release supporting Python 3.7, prepared this morning. We can move forward here.